packages/core/src/workflow/piece-buffering.js

```diff
@@ -10,14 +10,14 @@ import { decode as decodePiece } from '../data/piece.js'
  * @param {object} props
  * @param {import('@web3-storage/filecoin-api/types').Store<Data>} props.storeClient 
  * @param {import('@web3-storage/filecoin-api/types').Queue<Data>} props.queueClient
- * @param {string[]} props.pieceRecords 
- * @param {string} [props.groupId]
+ * @param {string[]} props.pieceRecords
+ * @param {boolean} [props.disableMessageGroupId] - only supported in FIFO queues
  */
 export async function bufferPieces ({
   storeClient,
   queueClient,
   pieceRecords,
-  groupId
+  disableMessageGroupId,
```

Comment:
Unfortunately, the existing docker image we can use for testcontainers to mimic a real Queue is an `elasticmq` which using dynamo client to create a queue could not make it a FIFO, so that we can use message group ID. Therefore, we need to allow not using it